app/views/templates/Layout_psp.scala.html

```diff
@@ -87,7 +87,6 @@
         phaseBanner = Some(standardBetaBanner(url = appConfig.betaFeedbackUnauthenticatedUrl)),
         userResearchBanner = Some(UserResearchBanner(url="/manage-pension-schemes/user-research-banner-psp"))
     )),
-    cspNonce = CSPNonce.get,
```

Comment:
We're still importing CSPNonce in this file, should we be removing this?

Reply:
removed unused imports